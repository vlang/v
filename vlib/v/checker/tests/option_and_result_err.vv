fn foo() !?i32 {
	return 6
}