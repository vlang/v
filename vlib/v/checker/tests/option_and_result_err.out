vlib/v/checker/tests/option_and_result_err.vv:1:11: error: the type must be Option or Result
    1 | fn foo() !?i32 {
      |           ^
    2 |     return 6
    3 | }
